apps/system-e2e/src/integration/web.spec.ts

```diff
@@ -28,4 +28,32 @@ describe('web', () => {
           .should('equal', '/'),
       )
   })
+
+  it('should have search results for common words', function () {
+    cy.visit('/')
+      .get('[data-testid="search-box"]')
+      .click()
+      .type('covid{enter}')
+      .get('[data-testid="search-result"]', { timeout: 15000 })
+      .as('test-results')
+      .should('have.length.at.least', 10)
+      .location('pathname', { timeout: 5000 })
+      .as('searchPath')
+      .get('@test-results')
+      .first()
+      .click()
+      .location('pathname', { timeout: 5000 })
+      .should((currentPath) => {
+        expect(currentPath).to.not.eq(this.searchPath)
+      })
```

Comment:
Suggestion?
```suggestion
      .should('not.eq', '@searchPath')
```

Reply:
I tried this and it did not work. Does it work for you?

Reply:
This did not work for me when I tried it. It compares to the literal value of '@searchPath'.